
> chat@1.0.0 dao C:\Users\gonzalo.bide\Repos\fizzmod\chat
> node db-server/dao.js

C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Parser.js:80
        throw err; // Rethrow non-MySQL errors
        ^

Error: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Handshake.Sequence._packetToError (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\sequences\Handshake.js:124:18)
    at Protocol._parsePacket (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:210:5)
    at addChunk (_stream_readable.js:308:12)
    at readableAddChunk (_stream_readable.js:289:11)
    --------------------
    at Protocol._enqueue (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\gonzalo.bide\Repos\fizzmod\chat\node_modules\mysql\lib\Connection.js:118:18)
    at Object.<anonymous> (C:\Users\gonzalo.bide\Repos\fizzmod\chat\db-server\dao.js:52:5)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'ER_NOT_SUPPORTED_AUTH_MODE',
  errno: 1251,
  sqlMessage: 'Client does not support authentication protocol requested by server; consider upgrading MySQL client',
  sqlState: '08004',
  fatal: true
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chat@1.0.0 dao: `node db-server/dao.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chat@1.0.0 dao script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\gonzalo.bide\AppData\Roaming\npm-cache\_logs\2019-11-20T10_58_28_315Z-debug.log

> chat@1.0.0 server C:\Users\gonzalo.bide\Repos\fizzmod\chat
> node app/server.js

events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (net.js:1300:14)
    at listenInCluster (net.js:1348:12)
    at Server.listen (net.js:1436:7)
    at Object.<anonymous> (C:\Users\gonzalo.bide\Repos\fizzmod\chat\app\server.js:14:8)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11
Emitted 'error' event on Server instance at:
    at emitErrorNT (net.js:1327:8)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'listen',
  address: '::',
  port: 8080
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chat@1.0.0 server: `node app/server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chat@1.0.0 server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\gonzalo.bide\AppData\Roaming\npm-cache\_logs\2019-11-20T10_58_37_319Z-debug.log
